src/assets/CollectibleDetectionController.ts

```diff
@@ -353,10 +365,7 @@ export class CollectibleDetectionController extends BaseController<
           }
 
           /* istanbul ignore else */
-          if (
-            !ignored &&
-            requestedSelectedAddress === this.config.selectedAddress
```

Comment:
Interesting, so this check was only added on the `CollectibleDetectionController` and not in the `CollectiblesController`